README.md

```diff
@@ -1 +1,25 @@
 # Badges Initiative
+
+This is the organizational repository for the Node.js Badges Strategic Initiative, as chartered under the [Community Committee].
+
+This initiative is tasked with facilitating the design, development and production of Node.js Open Badges and associated content.
```

Comment:
can you include a definition of what `Open Badges` is? I see some context with responsibilities below, but as someone unfamilliar with this initiative it is not obvious just yet

Reply:
Will do! The inaugural issue that inspired this new repo is here: https://github.com/nodejs/community-committee/issues/212

I'm planning to rework a lot of the info that lives in that issue into the repo over the next day or so – this PR is just to kick off the new initiative. Will update so it is more obvious to fresh eyes in the mean time though 🙂

Reply:
Rad thx, no need to block. Just wanted to share how this read without context

Reply:
I agree with @MylesBorins. Can you please add a description _why_ are we doing a open badges? Why are these important for our community?